### PR TITLE
fix(ci): Fix for issue 10258, rename NR and RPR file names

### LINKED ISSUE #10258: Rename PR artifacts propagating PR number and reverted PR information
Currently those artifacts are named NR and RPR, just renamed those to pr_number and is_reverted_pr

### PR DESCRIPTION
**Pull request by ranj8work**
Signed-off-by: Ranjini Iyer <ranj8work@gmail.com>

## Summary
Fix for issue [10258](https://github.com/magma/magma/issues/10258) to rename the NR and RPR file names under PR artifacts.

## Test Plan
Testing is done in my forked repo under master and test branch.

## Additional Information

- [NO] This change is backwards-breaking

### CODE REVIEW

File: .github/workflows/dco-check.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
         if: always()
         run: |
           mkdir -p ./pr
-          echo -n ${{ steps.reverted_pr_check.outputs.is_reverted_pr }} > ./pr/RPR
+          echo -n ${{ steps.reverted_pr_check.outputs.is_reverted_pr }} > ./pr/s_reverted_pr
```
Comment (quentinDERORY) [inline .github/workflows/dco-check.yml]: typo here

**Review by quentinDERORY — CHANGES_REQUESTED** [review]
Missing deploy_build_from_pr and build_all workflow

File: .github/workflows/dco-check.yml (hunk 1 above)
Comment (ranj8work) [reply .github/workflows/dco-check.yml]: Sorry, overlooked, Fixed it now @quentinDERORY

**Comment by ranj8work** [comment]
> Missing deploy_build_from_pr and build_all workflow

Was unaware of these workflows using PR artifacts. I made the changes in these workflows as well now. Thank you for letting me know.

**Comment by ranj8work** [comment]
@quentinDERORY - Addressed the comments and pushed the changes. Please review.

**Review by quentinDERORY — APPROVED** [review]

**Pull request merged**